simplekv/cache.py

```diff
@@ -110,6 +110,26 @@ def open(self, key):
             # cache error, ignore completely and return from backend
             return self._dstore.open(key)
 
+    def _copy(self, source, dest):
+        """Implementation of :meth:`~simplekv.CopyMoveMixin.copy`.
+        
+        Copies the data in the backing store and removes the destination key from the cache,
+         in case it was already populated.
+         """
+        k = self._dstore._copy(source, dest)
+        self.cache.delete(dest)
```

Comment:
No, I think using the private methods here is fine, given your other suggestion of moving the default implementations of _copy and _delete into the CopyMoveMixin.
You still have to use CopyMoveMixin with the CacheDecorator, and if you then hit a backend which actually supports a native _copy/_move, you can use it. Otherwise, you will get an exception anyways.

I agree that this discussion would be different if we would keep _copy/_move in KeyValueStore.